### ISSUE TITLE
Printing and Filterable

### ISSUE THREAD

**Issue by Emre10** [NonFactors/MVC6.Grid#81]
Hi,
My English is bad, so i'm sorry.
I have many suggestions about Grid.

1-) Will be printing pretty good? For example 

` columns.Add(model => $"<a title='Düzenle' href='/soforler/Duzenle/" + model.Sofor_NO + "'" + "><span class='fa fa-edit'></span></a> <a title='Detaylar' href='javascript: ShowWindow(" + model.Sofor_NO + ");'><span class='fa fa-list'></span></a> <a  href='#' title='Sil' onclick='Delete(" + model.Sofor_NO + ")' data-toggle='modal' data-target='#modal-default'><span class='fa fa-recycle'></span></a>").Encoded(false).Titled("İşlemler");`

when i click to CTRL+P, these links and filter image(rather font) are viewing in print page.

2-) 
`columns.Add(model => model.transactionDate.ToString("dd/MM/yyyy")).Titled("Date");` 
is not filterable.

Thank you for Grid :)

**Comment by Muchiachio** [comment]
1) Could you rephrase the question? I'm not sure what are you asking or what do you want to achieve here? If you can't, you could try using your native language and translate it using google translate, it might be more understandable.

2) You should format input by using `.Formatted` function:
`columns.Add(model => model.transactionDate).Titled("Date").Formatted("{0:dd/MM/yyyy}")`

**Comment by Emre10** [comment]
http://imgur.com/a/u8zTS

**Comment by Muchiachio** [comment]
You can hide them using css `@media print` directives.

**Comment by Emre10** [comment]
Thanks. I agree both of them
You are helpful
